tempesta_fw/t/functional/helpers/tf_cfg.py

```diff
@@ -108,4 +109,11 @@ def dbg(level, *args, **kwargs):
 if not skip_check:
     cfg.check()
 
+if os.geteuid() != 0:
+    TestFrameworkCfg.error("Tests must be run as root.")
+
+# the default value of fs.nr_open
+nofile = 1048576
+resource.setrlimit(resource.RLIMIT_NOFILE, (nofile, nofile))
```

Comment:
As per above, I assumed you wanted to preserve the possibility of launching testcases as standalone programs.